artifacttemplates/https%3A%2F%2Fust-quantil.github.io%2Fartifacttemplates/Braket-App_Start-w1/files/braket-app_start.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+echo 'Port:'$Port
+export AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID
+export AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
+export AWS_REGION=$AWS_REGION
+
+nohup python3 braket_app/adapter.py $Port > braket_app/output.log &
```

Comment:
If you add something like [here](https://github.com/OpenTOSCA/tosca-definitions-common/blob/main/artifacttemplates/http%253A%252F%252Fopentosca.org%252Fartifacttemplates/SpringWebApp_Start-w1/files/start.sh#L4) debugging becomes easier as 
1. the error output is also redirected to the log by adding `2>&1`
2. the process id of the started application is logged by performing `echo $! > braket-app.pid`